mailpoet/assets/css/src/components-automation-listing/_listing.scss

```diff
@@ -8,8 +8,12 @@
   .mailpoet-automation-listing {
     /* Prevent border radius beneath tabs */
     border-radius: 0 0 1px 1px;
-    /* Prevent overlapping box shadow since listing is rendered inside tabs */
-    clip-path: inset(-1px 0 0 0);
+  }
+}
+
+.mailpoet-automation-ellipsis-menu {
+  .woocommerce-ellipsis-menu__popover {
+    opacity: 1;
```

Comment:
I can't figure out why the popover is getting an opacity of `0` and an incorrect position calculated when I have the wordpress components CSS enqueued, so this is a temporary workaround to make the dropdown at least visible while I test it. I could really use some advice on a proper fix though.